modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -201,12 +201,14 @@ public void processOpts() {
         additionalProperties.put(CodegenConstants.PACKAGE_NAME, packageName);
         additionalProperties.put(CodegenConstants.PACKAGE_VERSION, packageVersion);
 
+        // NOTE: The null case can happen during PythonClientOptionsTest.checkOptionsProcessing().
+        String packagePath = packageName == null ? "" : packageName.replace('.', File.separatorChar);
```

Comment:
Would be cleaner to have packageName != null during the unit test `PythonClientOptionsTest.checkOptionsProcessing` ?

Reply:
Yes, that would be cleaner, but due to the slightly obscure mocking Kung Fu in the test I was unable to come up with something better.

I propose to land this change as it is (perhaps add a TODO), the current module name handling is totally unusable in non-trivial existing Python projects. Alternatively, somebody could give me a concrete hint how to fix the test... :wink: